user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataMain.java

```diff
@@ -80,18 +74,21 @@ public class RawDataMain extends Main implements BeforeEnterObserver {
   private final String rawDataSourceURL;
   private final String documentationUrl;
   private final ProjectInformationService projectInformationService;
+  private final ClientDetailsProvider clientDetailsProvider;
+  private final AsyncProjectService asyncProjectService;
   private transient Context context;
 
-  public RawDataMain(@Autowired RawDataDetailsComponent rawDataDetailsComponent,
+  public RawDataMain(
       @Autowired RawDataDownloadInformationComponent rawDataDownloadInformationComponent,
       @Autowired ExperimentInformationService experimentInformationService,
       @Autowired MeasurementService measurementService,
       @Autowired RemoteRawDataService remoteRawDataService,
       @Value("${server.download.api.measurement.url}") String dataSourceURL,
       @Value("${qbic.communication.documentation.url}") String documentationUrl,
-      @Autowired ProjectInformationService projectInformationService) {
+      @Autowired ProjectInformationService projectInformationService,
+      ClientDetailsProvider clientDetailsProvider, AsyncProjectService asyncProjectService,
```

Comment:
I am not sure, that the clientDetailsProvider here is thread safe?

Reply:
When looking at the client details provider I am usure how it is used and where the client details are set. As this is introduced as a spring component do you know if the client is always the correct one?
Probably this is out of the scope of this PR?